README.md

```diff
@@ -28,6 +28,7 @@ itteration of [widgetastic.patternfly4](https://github.com/RedHatQE/widgetastic.
 ### Components:
 - [alert](https://www.patternfly.org/components/alert)
 - [breadcrumb](https://www.patternfly.org/components/breadcrumb)
+- [calendar-month](https://www.patternfly.org/components/calendar-month)
```

Comment:
https://www.patternfly.org/components/date-and-time/calendar-month ^^

so better to categorize this just like we did for others eg. `menus`,  `forms`
same, we can create separate dir under component for src and tests.

Reply:
This should be fixed now